demos/demo.html

```diff
@@ -1,29 +1,33 @@
 <!DOCTYPE html>
 <html lang="en">
-	<head>
-		<title>demo</title>
-		<link rel="stylesheet" href="/public/main.css">
-	</head>
-	<body>
-		<div class="o-grid-container o-grid-container--snappy">
-			<div class="o-grid-row">
-				<h1>n-newsletter-signup demo</h1>
-			</div>
-			<div class="o-grid-row">
-				<div data-o-grid-colspan="12 S6 M4 L3">
-					{{> n-newsletter-signup/templates/newsletter flags=@root.flags}}
-				</div>
-			</div>
 
+<head>
+	<title>demo</title>
+	<link rel="stylesheet" href="/public/main.css">
+</head>
+
+<body>
+	<div class="o-grid-container o-grid-container--snappy">
+		<div class="o-grid-row">
+			<h1>n-newsletter-signup demo</h1>
+		</div>
+		<div class="o-grid-row">
```

Comment:
**Change request:** This results in nested `o-grid-rows` which I don't believe you can do. The nested one is on [line 23](https://github.com/Financial-Times/n-newsletter-signup/pull/107/commits/e10cbce67c077ec62d4532ab51b4bbafb6f43bfe#diff-7ced4bcd2a7e995789ee3c6d8459095a96ed232b6944f01702dac58842f16335R23)

It also results in the `h2` with the `Simple inline` text not rendering in the correct place

![Screenshot 2024-02-20 at 10 27 34](https://github.com/Financial-Times/n-newsletter-signup/assets/524573/f2be9c2b-3b1f-41c3-b6e5-577ebcb9b708)

Reply:
I have fix the missing closing div tag.